### PR TITLE
Document WFS3 sorting

### PR DESCRIPTION
**Pull request by elpaso**
Fixes: https://github.com/qgis/QGIS-Documentation/issues/5764

### CODE REVIEW

**Comment by DelazJ** [comment]
@elpaso /me trying to avoid the back-and-forth of the suggestion system... if you are OK

**Comment by elpaso** [comment]
@DelazJ absolutely! Thank you!

File: docs/user_manual/working_with_ogc/server/services.rst [hunk 1]
```diff
@@ -1878,6 +1878,23 @@ filters all features where attribute ``name`` ends with "value"
 
     http://localhost/qgisserver/wfs3/collection_one/items.json?attribute_one=*value
 
+Feature sorting
+---------------
+
+When querying for features it is possible to order the result set by field value using
+``sortby`` query parameter.
```
Comment (havatv) [inline docs/user_manual/working_with_ogc/server/services.rst:1885]:
```suggestion
It is possible to order the result set by field value using the ``sortby``
query parameter. 
```

File: docs/user_manual/working_with_ogc/server/services.rst [hunk 2]
```diff
@@ -1878,6 +1878,23 @@ filters all features where attribute ``name`` ends with "value"
...
+
+When querying for features it is possible to order the result set by field value using
+``sortby`` query parameter. 
+
+The results are sorted in ascending order by default. To sort the results in descending
+order a boolean flag ``sortdesc`` can be set:
+
+URL example:
+
+sort features by ``name`` in descending order:
```
Comment (havatv) [inline docs/user_manual/working_with_ogc/server/services.rst:1892]:
```suggestion
The results are sorted in ascending order by default.
To sort the results in descending order, a boolean flag (``sortdesc``) can be set
(URL example):
```

Comment (elpaso) [reply docs/user_manual/working_with_ogc/server/services.rst:1892]: Looks good, just check if `(URL example):` is in parenthesis in the rest of the document.

Comment (havatv) [reply docs/user_manual/working_with_ogc/server/services.rst:1892]: > Looks good, just check if `(URL example):` is in parenthesis in the rest of the document.

I think we can remove *(URL example)*

**Comment by havatv** [comment]
@elpaso, I applied the suggestion, so that we are ready to merge. Hope that is OK.

**Comment by elpaso** [comment]
LGTM, thank you!

**Pull request merged**